t/plugin/limit-count-redis-cluster.t

```diff
@@ -434,3 +434,70 @@ passed
 GET /hello
 --- response_body
 hello world
+
+
+
+=== TEST 12: set route, with rejected_msg
```

Comment:
Since this feature is not relative to the policy, we only need to test it in the limit-count.t instead of testing it in all three policies.

Reply:
okay